tooling/ipns/ipns_test.go

```diff
@@ -0,0 +1,64 @@
+package ipns
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var YEAR_100 time.Time
+
+func init() {
+	YEAR_100 = time.Time(time.Date(2123, time.March, 01, 01, 0, 0, 801257000, time.UTC))
```

Comment:
Should we check the current year and add 100?

Reply:
Nope, we're testing a static fixture, the ipns key is valid in March 2023 + 100 years, that's what we unit-test.